Information/CenterOfMass.FCMacro

```diff
@@ -589,8 +622,8 @@ class CenterofmassWidget(QtGui.QWidget):
     def store_prefered_units(self, txt):
         """Get units set as preference"""
         self.unitForD_text = txt
-        self.unitForD = self.unitForD_text.replace('³', '^3')         # density
-        self.unitForL = self.unitForD_text.split('/')[1].rstrip('³')  # length
+        self.unitForD = self.unitForD_text.replace('Â³', '^3')         # density
```

Comment:
I see a strange sign in Github (`Â`). Is it normal?

Reply:
It look like passing through different os the ³ changes in Â³

Reply:
I'm sorry but I don't believe this. This is not a problem with the OS but with your editor. You should set the endocing to UTF-8 without BOM.

Reply:
You're right, it's the text editor ! (but some time I swtich between windows and linux and I also switch between text editors...)